docs/SRS/SRS.tex

```diff
@@ -1330,6 +1331,11 @@ \subsection{Reflection}
 	      \dots
 
 	      \textbf{Joy Xiao}
+	      The first approach to master team management would be to keep track of the tasks and deadlines and plan
+	      meeting agendas in advance. Another approach is to give adequate time for team members to complete their
+	      tasks and give extra time in case an unknown issue arises. This will help us keep on schedule so we don't
+	      miss a deadline. I chose this skill because I think it would be very useful to have when working at a
+	      company.
 
 	      \dots
```

Comment:
```suggestion

	      The first approach to master team management would be to keep track of the tasks and deadlines and plan
	      meeting agendas in advance. Another approach is to give adequate time for team members to complete their
	      tasks and give extra time in case an unknown issue arises. This will help us keep on schedule so we don't
	      miss a deadline. I chose this skill because I think it would be very useful to have when working at a
	      company.

```